src/pe_reports/metrics.py

```diff
@@ -154,24 +130,17 @@ def breach_details(self):
                 "number_of_creds": "Number of Creds",
             }
         )
-
         return breach_det_df
 
     def password(self):
         """Return total number of credentials with passwords."""
-        pw_creds_csg = len(
-            self.query_cyberSix_creds[self.query_cyberSix_creds["password_included"]]
-        )
-        pw_creds_hibp = len(
-            self.query_hibp_view[self.query_hibp_view["password_included"]]
-        )
-        return pw_creds_csg + pw_creds_hibp
+        pw_creds = len(self.creds_view[self.creds_view["password_included"]])
+        return pw_creds
```

Comment:
Might as well just consolidate these lines:
```suggestion
        return len(self.creds_view[self.creds_view["password_included"]])
```